mdx/parse.ts

```diff
@@ -285,6 +285,24 @@ function parseMaterials (model: Model, state: State, size: number): void {
                 }
             }
 
+            if (model.Version >= 1100) {
+                state.int32(); // hd flag
+                const textureCount = state.int32();
+
+                for (let j = 0; j < textureCount; j++) {
+                    const textureId = state.int32(); //layer_texture.id
+                    layer.CoordId = state.int32(); //slot
```

Comment:
This code is incorrect
Every iteration is overriding the same TextureID over and over again
Instead, second int32 value should used as index in map
So iterations would be
```
TextureIds[0] = 0
TextureIds[1] = 1
TextureIds[2] = 2
...
```